api/apps/api/src/migrations/api/1608149578000-EnablePostgis.ts

```diff
@@ -0,0 +1,25 @@
+import { Logger } from '@nestjs/common';
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { PostgreSQLUtils } from '@marxan-api/utils/postgresql.utils';
+
+export class EnablePostgis1608149578000 implements MigrationInterface {
+  async up(queryRunner: QueryRunner): Promise<any> {
+    if (await PostgreSQLUtils.version13Plus()) {
+      await queryRunner.query(`
+CREATE EXTENSION IF NOT EXISTS postgis;
```

Comment:
This will enable PostGIS in the apidb, but we don't need it there.

This should not be needed when running the services natively either... are you sure this is needed?

Reply:
There are some columns of type geometry, so this is needed.

Reply:
Which ones? pretty sure api.geo shouldn't have any (otherwise it shouldn't even boot 🤔 )

Reply:
![image](https://user-images.githubusercontent.com/973964/146727615-b5849f54-9f28-4635-bfc9-ee1f8d8358e7.png)

Reply:
`extent` was dropped , as well as `tr_GetBbox` I believe 🤔

Reply:
ok, but that means they still exist at some point. Perhaps there is an improvement opportunity here, where this can be refactored further down the road.

Reply:
good point. I have created https://vizzuality.atlassian.net/browse/MARXAN-1165 to take care of this.